packages/styles/scss/patterns/blocks/leadspace-block/_leadspace-block.scss

```diff
@@ -30,14 +30,21 @@
       }
     }
 
-    &__heading {
-      @include carbon--type-style('expressive-heading-05', true);
-    }
-
     .#{$prefix}--content-block {
       padding-top: 0;
       padding-bottom: $layout-05;
 
+      &__heading {
+        color: $text-01;
+      }
+
+      &__copy {
+        color: $text-01;
+        p {
+          @include carbon--type-style('expressive-heading-03', true);
+        }
+      }
+
```

Comment:
@guilhermelMoraes Just confirming, but I think these changes have already been made in [this PR](https://github.com/carbon-design-system/ibm-dotcom-library/pull/2712/commits/8ee29fa833d588b39c4a5f59661fe360374e44f9#diff-6c2d7f4ee5a9324ff588e92bb9a91337R35)? It's the same token change to `ContentBlock`, which is what `LeadspaceBlock` is using.

Reply:
If we're setting the color token already in `ContentBlock`, perhaps the only change needed is the expressive heading token?

Reply:
Yeah, I think so. I've done this update before the #2712, but I'm going to change that since it's affecting only the `LeadspaceBlock`